dist/s_clang-format

```diff
@@ -0,0 +1,38 @@
+#! /bin/sh
+
+# Installation of the clang development package isn't standard, list a
+# couple of the places we're using.
+export PATH=$PATH:/usr/local/clang50/bin:/usr/local/llvm-devel/bin
```

Comment:
Why is this `clang50`, when we required Clang 6.0?

Reply:
8493ae7
Fixed.